frontend_lib/test/helper.spec.js

```diff
@@ -139,4 +144,24 @@ describe('helper.js', () => {
       expect(hasSpaces('bad Username')).to.eq(true)
     })
   })
+
+  describe('FETCH_CONFIG object', () => {
+    it('should include tracim client token header', () => {
+      expect('X-Tracim-ClientToken' in FETCH_CONFIG.headers).to.eq(true)
+      expect(FETCH_CONFIG.headers['X-Tracim-ClientToken']).to.be.a('string')
+    })
+
+    it('should store the client token in window session', () => {
+      expect(window.sessionStorage.getItem('tracimClientToken')).to.eq(FETCH_CONFIG.headers['X-Tracim-ClientToken'])
```

Comment:
to be sure, here, sessionStorage exists because, in the previous test, you called FETCH_CONFIG which called COMMON_REQUEST_HEADERS which called getOrCreateSessionClientToken which put it in the sessionStorage.
right ?
If I'm right, I think it makes the tests dependent on each other and it would be preferable not to

Reply:
The code putting the client token in the session storage is executed during the import of `helper.js` so I don't think we can say that the two tests are interdependent. I added this one because I didn't find a way to test that, even if you import `helper.js` several times the client token is the same.

Reply:
okok good
you can resolve after read